Configuration/NodeTypes.yaml

```diff
@@ -256,6 +257,36 @@
           group: options
 
 
+'CRON.FormBuilder:CheckBoxEnhanced':
+  superTypes:
+    'TYPO3.Neos:Content': true
+    'CRON.FormBuilder:CheckBox': true
+  ui:
+    label: CRON.FormBuilder:NodeTypes.Plugin:fields.checkbox.enhanced
+  properties:
+    text:
+      type: string
+      ui:
+        label: Text
+        inlineEditable: true
+        aloha:
+          placeholder: CRON.FormBuilder:NodeTypes.Plugin:fields.checkbox.enhanced.text
+          format:
+            'p': true
+          link:
+            'a': true
+    mainText:
+      type: string
+      ui:
+        label: CRON.FormBuilder:NodeTypes.Plugin:fields.checkbox.enhanced.text.dsgvo
```

Comment:
@Magarif this feature is no DSGVO specific, it's just a regular "checkbox with rich text" attached to it. We will just use it in our case for the DSGVO purposes, but it is not limited to this. So rename the labels and translations where you mention "dsgvo" into something more generic.